language-support/ts/daml-ledger/index.ts

```diff
@@ -16,15 +24,31 @@ export type CreateEvent<T extends object, K = unknown, I extends string = string
   payload: T;
 }
 
+/**
+ * An archived contract.
+ *
+ * @typeparam T The contract template type.
+ * @typeparam I The contract id type.
+ */
 export type ArchiveEvent<T extends object, I extends string = string> = {
   templateId: I;
   contractId: ContractId<T>;
 }
 
+/**
+ * An event is either the creation or archival of a contract.
+ *
+ * @typeparam T The contract template type.
+ * @typeparam K The contract key type.
+ * @typeparam I The contract id type.
+ */
 export type Event<T extends object, K = unknown, I extends string = string> =
   | { created: CreateEvent<T, K, I> }
   | { archived: ArchiveEvent<T, I> }
 
+/**
+ * Decoder for a [[CreateEvent]].
+ */
```

Comment:
This function is not exported. Does it show up in the generated docs or do we need to do something to hide it explicitly? Same in many places below.